docs/index.md

```diff
@@ -251,7 +251,7 @@ Example:
 def jwt_response_payload_handler(token, user=None, request=None):
     return {
         'token': token,
-        'user': UserSerializer(user).data
+        'user': UserSerializer(user,context={'request': request},).data
```

Comment:
Sorry.  I should have looked at the convention you were using for trailing comma before using my convention.  Thanks for taking the PR!